vMenu/LanguageManager.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MenuAPI;
+using Newtonsoft.Json;
+using CitizenFX.Core;
+using static CitizenFX.Core.UI.Screen;
+using static CitizenFX.Core.Native.API;
+using static vMenuClient.CommonFunctions;
+using static vMenuShared.ConfigManager;
+using static vMenuShared.PermissionsManager;
+
+namespace vMenuClient
+{
+    public class LanguageManager : BaseScript
+    {
+        public string Get (string originalText) {
+            
+            string transText = originalText;
+
+            if (!MainMenu.DumpedData.ContainsKey(originalText))
+            {
+                MainMenu.DumpedData.Add(originalText, "");
+            }
+
+            if (MainMenu.CurrentLanguage.ContainsKey(originalText)) {
+                try {
+                    transText = (string)MainMenu.CurrentLanguage[originalText];
```

Comment:
This try / catch is not needed, you've already checked if the text is in there so nothing should raise an exception here.